verilog/formatting/tree_unwrapper.cc

```diff
@@ -896,6 +921,22 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeLeaf& leaf) {
   // up to a newline.
   LookAheadBeyondCurrentLeaf();
 
+  const std::regex uvm_end("^`uvm_.*_end$", std::regex_constants::basic);
```

Comment:
Suggest simplifying to just use `absl::StartsWith` and `EndsWith`, which is much faster than pattern matching.

Reply:
Done